README.md

```diff
@@ -4,13 +4,9 @@
 
 In this repository, we apply the [phenotypic profiling model](https://github.com/WayScience/phenotypic_profiling_model), which predicts phenotypic class of single cells using nuclei features, to the [JUMP-Target pilot data](https://github.com/jump-cellpainting/JUMP-Target) from the [JUMP consortium](https://jump-cellpainting.broadinstitute.org/).
 
-In this dataset, there are 51 plates that are associated with one of three treatment types:
+In this dataset, there are 51 plates with one of three perturbation types (Clustered Regularly Interspaced Short Palindromic Repeats [CRISPR], Open Reading Frame [ORF], and Compound) for two cell lines (A549 and U2OS).
 
-- Compound
-- CRISPR
-- ORF
-
-Each treatment has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
+Each perturbation type has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
```

Comment:
```suggestion
Each perturbation type has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
We accessed this reference data from the JUMP `experimental-metadata.tsv` file located [here](https://github.com/jump-cellpainting/2024_Chandrasekaran_NatureMethods/blob/a18fd7719c05b638c731142b0d42a92c645e2b33/benchmark/output/experiment-metadata.tsv).
```

Reply:
True?

Reply:
NVM! I see that you added this detail in the specific section's README. Feel free to ignore this